### DISCUSSION TITLE
How do I match from  inside a react component or html tag?

### DISCUSSION THREAD

**Discussion by adriangalilea** [helix-editor/helix#10137] [Q&A] [answered]
Is this possible?
```
<Foo>ba|cursor|r  bar bar bar bar</Foo> 
```
I want to select from `>` to `<` so it would be `bar  bar bar bar bar`

Anyone?

Thank you!

**Comment by CptPotato** [comment, marked as answer]
There's currently no function to match inside tags like `miw` or `mi"`, but you can try using "expand selection" <kbd>Alt</kbd> <kbd>o</kbd>.

**Reply by adriangalilea** [reply]
Right, forgot about <kbd>alt</kbd> <kbd>o</kbd>
Thanks!
